salt/states/lxc.py

```diff
@@ -290,13 +299,19 @@ def present(name,
     return ret
 
 
-def absent(name):
+def absent(name, stop=False)
     '''
     Ensure a container is not present, destroying it if present
 
     name
         Name of the container to destroy
 
+    stop
+        stop before destroying
+        default: false
+
+        .. versionadded:: 2015.5.0
```

Comment:
should be 2015.5.2 (2015.5.1 was tagged internally today, so this kwarg won't be in a release until 2015.5.2)